### PR TITLE
Cloud service information

### PR DESCRIPTION
**Pull request by schaefi**
Added compute vm show command
    
show detailed information about all virtual machines running in the specified cloud service
https://trello.com/c/PDvqeQZN/168-azurectl-next

### CODE REVIEW

**Comment by schaefi** [comment]
@bear454 no idea why it complains about code duplication in the test, I also would prefer things like landscape, codacy and friends to provide as report and statistics but not cause a fail result. For that purpose we have the unit tests, the coverage and the flake8 checks, that imho should be enough

File: azurectl/commands/compute_vm.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
            [--instance-name=<name>]
            [--wait]
        azurectl compute vm regions
+       azurectl compute vm show --cloud-service-name=<name>
```
Comment (rjschwei) [inline azurectl/commands/compute_vm.py]: Does it makes sense, or is there even a way to provide info about a specific VM?

azurectl compute vm show --cloud-service-name

provides info about all the VMs in the cloud service and more and

azurectl compute vm show --cloud-service-name=... -vm-name=...

would provide information only about this specific machine within the cloud service?

Comment (schaefi) [inline azurectl/commands/compute_vm.py]: The sdk method allows only to set details=True and that will then provide the complete information. We could however think of reducing the information to the specified instance name

Comment (schaefi) [inline azurectl/commands/compute_vm.py]: However I don't see it as that important

**Comment by bear454** [comment]
@schaefi: sorry, I didn't realize I had set up codacy to comment here; I'll try to turn those off for now.

**Comment by bear454** [comment]
Looks good to me.

**Pull request merged**